scss/core/extensions/_utilities.scss

```diff
@@ -1,13 +1,12 @@
-@each $color, $value in $colors {
-  @include bg-variant(".bg-#{$color}", $value);
-}
-
-@each $color, $value in $colors {
-  @include text-emphasis-variant(".text-#{$color}", $value);
-}
+// Add background, border and text color utilities
```

Comment:
How do you feel about the Oxford comma?  _ducking & running_

Reply:
ducking, running, & yelling! ;)